src/main/java/teammates/ui/webapi/output/SearchStudentsStudentData.java

```diff
@@ -0,0 +1,42 @@
+package teammates.ui.webapi.output;
+
+import javax.annotation.Nullable;
+
+import teammates.common.datatransfer.attributes.StudentAttributes;
+
+/**
+ * Contains attributes for the Students search response object.
+ */
+public class SearchStudentsStudentData extends CommonSearchUserData {
+    private final String team;
+    private final String section;
+    private final JoinState joinState;
+
+    @Nullable
+    private final String comments;
+
+    public SearchStudentsStudentData(StudentAttributes studentAttributes) {
+        super(studentAttributes.getName(), studentAttributes.getEmail(), studentAttributes.getCourse(),
+                studentAttributes.getGoogleId());
+        this.comments = studentAttributes.getComments();
+        this.team = studentAttributes.getTeam();
+        this.section = studentAttributes.getSection();
+        this.joinState = studentAttributes.isRegistered() ? JoinState.JOINED : JoinState.NOT_JOINED;
```

Comment:
Think JoinState can be moved to the common class even though it is not used now, reason being just to standardize attributes that can be found in both `StudentData` and `InstructorData` now.

Reply:
Maybe can also rename the SearchStudentsStudentData to SearchStudentsResult and this file to SearchStudentData to make the file name easier to read. but its not a big deal

Reply:
@hcwong you mean rename `SearchStudentsStudentData` -> `SearchStudentData` and `SearchStudentsData` -> `SearchStudentsResult` right?

Reply:
yea